zyngui/zynthian_gui_stepsequencer.py

```diff
@@ -1040,6 +1042,10 @@ def register_zyncoder(self, encoder, object):
 			pin_a=zynthian_gui_config.zyncoder_pin_a[encoder]
 			pin_b=zynthian_gui_config.zyncoder_pin_b[encoder]
 			zyncoder.setup_zyncoder(encoder, pin_a, pin_b, 0, 0, None, 64, 128, 0)
```

Comment:
The last parameter on zyncoder.setup_encoder is the step size. When 0 is passed, dynamic step size (acceleration) is enabled.